### PR TITLE
Fix composer json/lock

### PR DESCRIPTION
**Pull request by Alex-D**
This patch allow this package to be published to packagist
or to be used as a repository in your own composer.json

Please after merging that, loggin to https://packagist.org/ and publish this package by simply go to https://packagist.org/packages/submit and paste this repository URL.

Thanks :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+    "name": "allegro/php-protobuf",
+    "description": "Google's Protocol Buffers for PHP",
+    "type": "library",
+    "keywords": [
+        "protocol buffers",
+        "data",
+        "pb"
+    ],
+    "homepage": "https://github.com/allegro/php-protobuf",
+    "bin": ["bin/protoc"],
```
Comment (hjagodzinski) [inline composer.json]: The `bin/protoc` doesn't exist. If anything the `protoc-gen-php.php` should be added as a vendor binary. Simple listing it here doesn't work.

**Comment by Alex-D** [comment]
You right, I just drop this useless line.

**Comment by hjagodzinski** [comment]
Thanks! I'm not able to submit to packagist right now. I will do it next week.

**Comment by hjagodzinski** [comment]
@Alex-D you can find now [the package](https://packagist.org/packages/allegro/php-protobuf) in the packagist.

**Comment by Alex-D** [comment]
Yeah! Cool :)
GJ!

**Comment by hjagodzinski** [comment]
Thanks! :)

**Pull request merged**